cvmfs/magic_xattr.cc

```diff
@@ -8,11 +8,13 @@
 #include <string>
 #include <vector>
 
+#include "cache_posix.h"  // PosixCacheManager
```

Comment:
This type of comment is likely to go stale quickly, when another PR uses something from the header but does not update the comment, let's remove these.